### ISSUE TITLE
Can't login to Volta Desktop (Windows)

### ISSUE THREAD

**Issue by Pjiesco** [volta-net/feedback#26]
Trying to log-in to volta desktop with github brings you to this page
![image.png](https://volta.s3.fr-par.scw.cloud/image_3e1c63fa48.png)
Clicking on "Open Volta" doesn't open the app and logs to console:
```
Failed to launch 'volta://jwt=[TOKEN]' because the scheme does not have a registered handler.
```
(where [TOKEN] is the actual token)

**Comment by ChristopheCVB** [comment]
I already reported this and I think @smarroufin has already fixed it for next version

**Comment by benjamincanac** [comment]
Hello @Pjiesco,

This should now be fixed in latest [v1.4.0](https://volta.net/changelog/v1.4.0) version. You application should auto-update.

Let me know if you still encounter the issue.
